docs/dm/assets.md

```diff
@@ -87,62 +98,72 @@ This menu includes a plethora of options.
 
 ### Is a token
 
-Selecting this option classifies the asset as a token. 
-This will help you keep track of which assets are tokens and which are other assets.
+Selecting this option classifies the object as a token. 
+Tokens function fundamentally differently than other shapes.
+
+Tokens cannot pass through shapes with the 'blocks movement' option enabled. 
+Tokens are also effected by lighting.
+Check the [lighting and vision](/docs/dm/light-shadows/) document for more information on how lighting works in relation to tokens.
 
```

Comment:
It just occured to me and it might be helpful for players: Instead of deleting user-controlled, but incapacitated tokens (e.g. familiars, animal companions, summoned monsters etc.) or modifying their vision, they can simply be de-tokened, so to say, in order to let the controlling player no longer benefit from its vision.

Maybe this can be formulated better/easier and included, here? :)